src/main/java/org/mitre/synthea/helpers/Concepts.java

```diff
@@ -50,7 +51,7 @@ public static void main(String[] args) throws Exception {
     
     Files.write(outFilePath, output, StandardOpenOption.CREATE);
```

Comment:
There's an existing bug here where if you don't have the output directory (ex, if you clone the repo and immediately run `gradle concepts`), it will throw a FileNotFoundException. We should probably also use the configuration property `exporter.baseDirectory`. I'll clean that up in a later PR.